### ISSUE TITLE
Change repository name?

### ISSUE THREAD

**Issue by icco** [github/game-off-2013#19]
I've changed my repository's name, and Github still lists it as a fork of this repo, but I wanted to make sure that didn't disqualify me or anything.

Thanks!

**Comment by leereilly** [comment]
Yes, you can change the repository name. As long as we can see the repository showing up in the list of forks, you're good to go. Thanks for taking part and good luck!!!

Cheers,
  Lee :beers:
